src/TestFramework/Coverage/CodeCoverageData.php

```diff
@@ -199,6 +193,18 @@ private function addTestExecutionInfo(array $coverage): array
         return $newCoverage;
     }
 
+    public function isLineFunctionSignature(string $filePath, int $line): bool
```

Comment:
Since this gets called multiple times it might be beneficial to 'cache' its results somehow.